cartridges/int_adyen_overlay/cartridge/scripts/util/AdyenHelper.ds

```diff
@@ -446,6 +454,24 @@ var __AdyenHelper: Object = {
         return {error: true};
     },
 
+    add3DS2Data: function(jsonObject){
+        jsonObject["additionalData"].allow3DS2 = true;
+        jsonObject["channel"] = "web";
+
```

Comment:
please add 'origin' to the request fields: https://github.com/Adyen/adyen-magento/blob/feature/PW-1146-3DS2/app/code/community/Adyen/Payment/Model/Api.php#L147